internal/build/boil_steps.go

```diff
@@ -0,0 +1,31 @@
+package build
+
+import (
+	"github.com/windmilleng/tilt/internal/model"
+)
+
+type BoiledStep struct {
+	cmd         model.Cmd
+	pathMapping pathMapping
```

Comment:
i don't understand what the goal of this pathMapping property is? afaict, it's never used

Reply:
Yeah looks like we don't need this anymore, I'll remove it.